### DISCUSSION TITLE
Can I build a portal like ransomwatch.telemetry.ltd

### DISCUSSION THREAD

**Discussion by Blason** [thetanz/ransomwatch#86] [General]
Hi,

I am pretty confuse with this repo. Does that mean I can build custom portal like [ransomwatch.telemetry.ltd](https://ransomwatch.telemetry.ltd/) or we can query to ransomwatch.telemetry.ltd using python scripts.

What is the docker image is being used for?

**Comment by joshhighet** [comment]
Errr, What are you trying to achieve?

Yes, You can poll from this dataset, refer to the end of the README for some basic examples

**Comment by Blason** [comment]
Got it. I was thinking it builds a portal like randomwatch instead docker
image is a socks proxy.
